packages/tokens/_build/style-dictionary.js

```diff
@@ -50,6 +50,34 @@ StyleDictionary.registerTransform({
   },
 });
 
+/**
+ * @function StyleDictionary.registerTransform()
+ * Defines a custom StyleDictionary transform.
+ *
+ * swisspost/px-to-rem:
+ * Used to transform font sizes from px to rem
+ * This ensures accessible font-sizes
+ */
+
+StyleDictionary.registerTransform({
+  name: 'swisspost/px-to-rem',
+  type: 'value',
+  filter: token => {
+    const usesDtcg = token.$type && token.$value;
+    return token[usesDtcg ? '$type' : 'type'] === PX_TO_REM_TOKEN_TYPE;
+  },
+  transform: token => {
+    const usesDtcg = token.$type && token.$value;
+    const baseFontSize = 16;
```

Comment:
Should we define this as a constant like the `PX_TO_REM_TOKEN_TYPE` in the constants.js file as well?
What do you think?